tests/phpUnit/Api/MediaLibraryApiTest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace Tests\Api;
+
+use App\Entity\MediaPackage;
+use App\Entity\MediaPackageCategory;
+use App\Entity\MediaPackageFile;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\EntityManager;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Symfony\Component\HttpFoundation\File\File;
+
+/**
+ * @internal
+ * @coversNothing
```

Comment:
https://phpunit.readthedocs.io/en/7.3/annotations.html#coversnothing

change it to covers, would be nice to have a coverage report at the end.